pkg/driver/driver_alicloud.go

```diff
@@ -112,16 +57,16 @@ func (c *AlicloudDriver) Create() (string, string, error) {
 		return "", "", err
 	}
 
-	request := c.createRunInstancesRequest()
+	request := ecs.CreateRunInstancesRequest()
 	//request.DryRun = requests.NewBoolean(true)
 
-	request.ImageName = c.AlicloudMachineClass.Spec.ImageName
+	request.ImageId = c.AlicloudMachineClass.Spec.ImageName
```

Comment:
We should rename our MachineAPI fields from ImageName to ImageID also to maintain consistency